lib/index.js

```diff
@@ -51,11 +56,15 @@ function lint(editor) {
     plugins.push(recommended);
   }
 
+  if (styleGuideWithoutConfig) {
+    plugins.push(styleGuide);
```

Comment:
The code currently adds all 3 included configs if a user customized one isn't defined. I'm assuming `remark` handles this?

I guess this is worded the same as the other two settings, so seeing them together in the settings makes the wording make sense.